tests/api/test_backups.py

```
@@ -718,6 +718,46 @@
+@pytest.mark.usefixtures("tmp_supervisor_data")
+async def test_upload_with_filename(
+    api_client: TestClient, coresys: CoreSys, filename: str, expected_status: int
+):
+    """Test uploading a backup to multiple locations."""
+    backup_file = get_fixture_path("backup_example.tar")
+
+    with backup_file.open("rb") as file, MultipartWriter("form-data") as mp:
+        mp.append(file)
+        import logging
+
+        logging.warning(f"Testing with {filename}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)